types/task.ts

```diff
@@ -1,3 +1,5 @@
+import { SubscriptionPricing } from "..";
```

Comment:
Please import from the full path, rather than `".."`

Reply:
changed